src/NUnitFramework/testdata/nunit.testdata.csproj

```diff
@@ -1,50 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\Common.props" />
   <PropertyGroup>
-    <AssemblyName>nunit.testdata</AssemblyName>
-    <RootNamespace>NUnit.TestData</RootNamespace>
     <TargetFrameworks>net20;net35;net40;net45;netcoreapp1.1</TargetFrameworks>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-    <OutputPath>..\..\..\bin\$(Configuration)\</OutputPath>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <DebugType Condition="'$(TargetFramework)' != '' AND '$(TargetFramework)' != 'netcoreapp1.1'">Full</DebugType>
-    <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>..\..\nunit.snk</AssemblyOriginatorKeyFile>
-    <DefineConstants>TRACE;ASYNC</DefineConstants>
+    <RootNamespace>NUnit.TestData</RootNamespace>
     <Optimize>false</Optimize>
```

Comment:
Only place where we have `<Optimize>false</Optimize>`

Reply:
It's required for this project because our tests examine the stacktrace. Perhaps we should add a comment here if VS will maintain it. Perhaps even a  condition that errors if it is changed.

Reply:
Good ideas. I can add a test that check's the assembly's JIT optimization flag.